superset/models/schedules.py

```diff
@@ -31,17 +31,20 @@
 
 
 class ScheduleType(str, enum.Enum):
+    # pylint: disable=invalid-name
```

Comment:
I opted not the change these enums (and others) to uppercase as the string representation of the lowercase name is used in the database and I wanted to minimize the size of the PR.